docs/pages/build/building-on-ci.md

```diff
@@ -156,26 +156,17 @@ version: 2.1
 executors:
   default:
     docker:
-      - image: circleci/node:16
+      - image: cimg/node:lts
```

Comment:
@byCedric do you prefer LTS (6 months after release) or to pin to e.g. 16.3?

Reply:
In this case, I don't think it matters much for the example, two thoughts:

1. Expo should always support the _latest_ LTS, that's what this `lts` tag is referring to. So, this could be a pro in terms of keeping the environment in sync with whatever we support.
2. If the project uses specific features or requirements that force them to an older `lts`, they probably are experienced enough to know that they should change this.

Reply:
And in case the workflow fails because the project is too old, I think this is a good reminder for the developers to start upgrading their SDK as well 😬